src/snap.py

```diff
@@ -0,0 +1,101 @@
+import os
+import sys
+import pandas as pd
+from typing import Any, Dict, Tuple, List
+sys.path.append(os.path.abspath(''))
+from src.census_response import county_fips  # noqa: E402
+
+
+def load_excel(src: str) -> Any:
+    table = pd.read_excel(src, sheet_name=None, engine='openpyxl')
+    for k in table:
+        # drop empty rows (i.e. all values are NaN)
+        table[k].dropna(axis=1, how='all', inplace=True)
+        # drop empty columns (i.e. any value in column is NaN)
+        table[k].dropna(axis=0, how='any', inplace=True)
+
+        # drop last row (contains column sums)
+        table[k] = table[k][table[k]['Unnamed: 0'] != 'Sum:']
+
+        # ensure we're indexing from 0
+        table[k].reset_index(inplace=True)
+
+        # drop column of old indices
+        table[k].drop(table[k].columns[0], axis=1, inplace=True)
+    return table
+
+
+def rename_columns(table: Dict[str, Any]) -> None:
+    for k in table:
+        columns = []
+        columns.append('county')
+        columns.append('race_white')
+        columns.append('race_black')
+        columns.append('race_native')
+        columns.append('race_asian')
+        columns.append('race_pacific')
+        columns.append('race_hispaniclatino')
+        columns.append('race_unknown')
+        table[k].columns = columns
+
+
+def add_fips_column(table: Dict[str, Any]) -> None:
+    fips = county_fips()
+    # We're going to make a separate dictionary where the keys are just
+    # the county name in all caps. Note that we're trimming out spaces
+    # as well to account for e.g. DE WITT vs. DEWITT
```

Comment:
What happens if you don't take out the spaces?

Reply:
If we don't trim the spaces, we'll get a missing key exception on De Witt county, which is also written as DeWitt.

Reply:
ouch that's annoying. "The U.S. Census Bureau and the USGS list the county's name as De Witt, although the county uses the name DeWitt (no space)." https://en.wikipedia.org/wiki/DeWitt_County,_Illinois